RAMP/packer.py

```diff
@@ -49,85 +50,97 @@ def archive(module_path, metadata, archive_name='module.zip'):
     """
     Archives both module and module metadata.
     """
-    archive_name = archive_name.format(**metadata)
-    with open('module.json', 'w') as outfile:
+    with tempfile.NamedTemporaryFile(mode='w', prefix='ramp.json', delete=False) as outfile:
+        jfile = outfile.name
+        archive_name = archive_name.format(**metadata)
         json.dump(metadata, outfile, indent=4, sort_keys=True)
 
-    archive_file = zipfile.ZipFile(archive_name, 'w', zipfile.ZIP_DEFLATED)
     try:
-        archive_file.write(module_path, metadata["module_file"])
-        archive_file.write('module.json')
-        print(archive_name)
+        with zipfile.ZipFile(archive_name, 'w', zipfile.ZIP_DEFLATED) as archive_file:
+            archive_file.write(module_path, metadata["module_file"])
+            archive_file.write(jfile, arcname='module.json')
+            print(archive_name)
```

Comment:
Do we want this print?

Reply:
Was there before. I think we use this output to find out what's the name of the archive is if we used expressions like {samantic-version} as part of the filename.